telegram/ext/_fsm/states.py

```diff
@@ -0,0 +1,114 @@
+"""This Module contains implementations of State classes for Finite State Machines"""
+
+import abc
+import contextlib
+from typing import ClassVar, Optional
+from uuid import uuid4
+
+
+class State(abc.ABC):
+    __knows_uids: ClassVar[set[str]] = set()
```

Comment:
Wait I must really misunderstand smth, how does this work across states

Reply:
uh … that's how class variables work? :D In the init `self.__knows_uids` is basically the same as `State.__knows_uids` …